APIs/getDailyWeather.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const router = express.Router();
+const axios = require("axios");
+
+const { getDailyWeatherResponse } = require("../DTOs/index");
+const { APPID , FORECAST_URL, weekday } = require("../common/index");
+
+const getCurrentHour = () => {
```

Comment:
this function name should be getHourDivisibleByThree and this function should be in common/functions.js so we could this function individually

Reply:
`
/**
* explain why we are doing
*/

let currentHour = hours;

const CurrentHourIsNotDivisibleByThree = remainder !== 0;

if(CurrentHourIsNotDivisibleByThree)
{
  const isCurrentHourNearestToNextHour = remainder > 1;
  if(isCurrentHourNearestToNextHour) currentHour += 1;
  else currentHour -= 1;
}
`